microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -38,227 +56,269 @@ To use experimental APIs, download a prerelease version of the WebView2 SDK from
-## Using an environment variable
+##### [Group policy](#tab/group-policy)
 
-To make your application use a Microsoft Edge preview channel by using an environment variable:
+Set the `BrowserExecutableFolder` policy.
 
-1. In the Windows search bar, enter "environment", and then select **Edit the system environment variables**.
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`
+Value: `<path to desired Microsoft Edge preview channel>`
 
-   ![Using the Windows search bar to find where to edit environment variables](./set-preview-channel-images/search-bar-edit-sys-env-vars.png)
+Do either of the following:
 
-1. In the **System Properties** dialog, select the **Advanced** tab, and then select the **Environment Variables** button.
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  See [Configure Microsoft Edge policy settings on Windows devices](/deployedge/configure-microsoft-edge).
 
-   ![The Environment Variables button in the System Properties dialog](./set-preview-channel-images/system-properties-env-vars.png)
+* Use the built-in policy on Intune.  See [Configure Microsoft Edge policy settings in Microsoft Intune](/mem/intune/configuration/administrative-templates-configure-edge).
+
+---
+
+
+<!-- ====================================================================== -->
+## FAQs
+
+
+<!-- ------------------------------ -->
+#### When does the new runtime get picked up?
+
+If you are using an environment variable, registry key, or group policy, they are only applied to any new processes that are created; they do not apply to processes that are already running.  To ensure that all processes use the environment variable, registry key, or group policy, you would need to restart the processes.
+
+
+<!-- ------------------------------ -->
+#### When using `ChannelSearchKind`, how do I make sure the right channel is picked up when I have multiple preview browser channels?
 
-1. In the **User variables** section of the **Environment Variables** dialog, select **New**.
+In addition to reversing the channel search order, you can modify the set of release channels, to provide further control over channel selection.  The loader will search for channels on the device according to the channel search kind, but it will only consider channels that are in the specified set of release channels.
 
-1. In the **New User Variable** dialog, set the **Variable name** to `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER`, and set the **Variable value** to the path to your preferred browser channel or fixed version binaries.
+This option is useful if you would like to target a particular channel, instead of simply choosing the least stable channel.  By default, release channels includes all channels: Stable (0), Beta (1), Dev (2), and Canary (3).
```

Comment:
```suggestion
This option is useful if you would like to target a particular channel, instead of simply choosing the most or least stable channel.  By default, the release channels includes all channels: Stable (0), Beta (1), Dev (2), and Canary (3).
```